test/xbgpu/test_engine.py

```diff
@@ -523,9 +544,9 @@ def verify_beam_sensors(
         Dictionary of lists of sensor updates. They dictionary keys are sensor
         names, the values are a list of readings for each sensor update captured
         via the callback attached to :class:`BPipeline` sensors.
-    first_timestamp, last_timestamp
-        Two timestamps indicating the start and end of data processing
-        by the :class:`BPipeline`.
+    runtime_timestamps
```

Comment:
What does "runtime" mean in this context?

Reply:
Yeah I was trying to think of a better adjective because it's
* the timestamps 'during the test', and 
* not the timestamp at the instant the test begins

Perhaps I'll rename to `beam_request_timestamps` (?)